omisego-sdk/src/main/java/co/omisego/omisego/network/ewallet/EWalletAPI.kt

```diff
@@ -49,4 +53,10 @@ interface EWalletAPI {
 
     @POST(RETRIEVE_TRANSACTION_REQUEST)
     fun retrieveTransactionRequest(@Body request: TransactionRequestParams): OMGCall<TransactionRequest>
+
+    @POST(Endpoints.CONSUME_TRANSACTION_REQUEST)
```

Comment:
You used static import for other constants :)

Reply:
I've already fixed it in the future PR (can't remember which one) 👍